### DISCUSSION TITLE
How to install Store app？

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/WindowsAppSDK#1445] [Q&A] [answered]
There are Windows like Windows LTSC version without Store app installed, how to fix it?

**Comment by riverar** [comment, marked as answer]
This is not relevant to the Windows App SDK. Try posting your question on [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/).

**Comment by HppZ** [comment]
https://docs.microsoft.com/en-us/answers/questions/324813/how-to-enable-the-app-installer-on-windows-10-ltsc.html
but your official reply is not a official solution.

**Comment by HppZ** [comment]
use a third-party script to fix this?
https://github.com/kkkgo/LTSC-Add-MicrosoftStore
